openjpa/pom.xml

```diff
@@ -129,48 +129,19 @@
                             </artifactSet>
                             <relocations>
                                 <relocation>
-                                    <pattern>javax.annotation.security</pattern>
-                                    <shadedPattern>jakarta.annotation.security</shadedPattern>
+                                    <pattern>javax.annotation</pattern>
+                                    <shadedPattern>jakarta.annotation</shadedPattern>
+                                    <excludes>
+                                        <exclude>javax.annotation.process.**</exclude>
+                                    </excludes>
                                 </relocation>
                                 <relocation>
-                                    <pattern>javax.annotation.Generated</pattern>
-                                    <shadedPattern>jakarta.annotation.Generated</shadedPattern>
-                                </relocation>
-                                <relocation>
-                                    <pattern>javax.annotation.PostConstruct</pattern>
-                                    <shadedPattern>jakarta.annotation.PostConstruct</shadedPattern>
-                                </relocation>
-                                <relocation>
-                                    <pattern>javax.annotation.PreDestroy</pattern>
-                                    <shadedPattern>jakarta.annotation.PreDestroy</shadedPattern>
-                                </relocation>
-                                <relocation>
-                                    <pattern>javax.annotation.Resource</pattern>
-                                    <shadedPattern>jakarta.annotation.Resource</shadedPattern>
-                                </relocation>
-                                <relocation>
-                                    <pattern>javax.annotation.Resources</pattern>
-                                    <shadedPattern>jakarta.annotation.Resources</shadedPattern>
-                                </relocation>
-                                <relocation>
-                                    <pattern>javax.el</pattern>
```

Comment:
these seems to be lost ...

Reply:
not used ;)

but guess it is worth a test ran after shade goal execution to validate the produced jar does not contain any javax.xxx which is unexpected (using asm since we have it in the cp)

Reply:
Indeed and on purpose. There is no use of EL in OpenJPA. So no need to have them in my opinion.